workers/data_refinery_workers/processors/array_express.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import, unicode_literals
+import os
+import zipfile
+from typing import Dict
+import rpy2.robjects as ro
+from celery import shared_task
+from celery.utils.log import get_task_logger
+from data_refinery_workers.processors import utils
+
+logger = get_task_logger(__name__)
+
+
+def cel_to_pcl(kwargs: Dict):
+    batch = kwargs["batch"]
+
+    temp_directory = utils.ROOT_URI + "temp/" + batch.internal_location
+    target_directory = utils.ROOT_URI + "processed/" + batch.internal_location
+    os.makedirs(temp_directory, exist_ok=True)
+    os.makedirs(target_directory, exist_ok=True)
+
+    raw_file_name = batch.download_url.split('/')[-1]
+    zip_location = (utils.ROOT_URI + "raw/" + batch.internal_location
+                    + raw_file_name)
+
+    # I need to figure out location tracking
+    zip_ref = zipfile.ZipFile(zip_location, 'r')
```

Comment:
What if `zip_location` doesn't exist in the file system? Or in the typical use case this issue should never happen?